src/CarouselDots/index.tsx

```diff
@@ -73,10 +73,10 @@ const CarouselDots = ({
   const refScrollView = useRef<ScrollView>(null);
   const [curIndex, setCurIndex] = useState<number>(currentIndex);
   const positiveMomentum = useRef<boolean>(false);
-  const prevIndex = usePrevious(curIndex, curIndex);
+  const prevIndex = usePrevious(curIndex, curIndex - 1 >= 0 ? curIndex - 1 : curIndex);
```

Comment:
Why do you need to set a prevValue that is the index minus 1? I mean, it would be 0 if the curIndex is 1